src/style/tokens/typography.css

```
@@ -1,36 +1,57 @@
-  --font-size-15: 0.9375rem;
-  --font-size-md: 1rem;
-  --font-size-lg: 1.125rem;
-  --font-size-19: 1.2rem;
-  --font-size-xl: 1.25rem;
-  --font-size-2xl: 1.5rem;
-  --font-size-28: 1.75rem;
-  --font-size-3xl: 2rem;
-  --font-size-4xl: 3rem;
+  /* ── Font families ────────────────────────────────────────────────────────── */
+  --font-family-base:
+    -apple-system, BlinkMacSystemFont, 'Inter', 'SF Pro Display', 'SF Pro Text',
```

Stylelint (17.6.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)